api/views/views.py

```diff
@@ -0,0 +1,34 @@
+from flask_restplus import Resource
+from flask import Flask, request, jsonify
+
+from api.models.models import Users
+from main import api
+
+from api.utilities.schemas.user_schema import UserSchema
+from api.utilities.messages.success_messages import SUCCESS_MESSAGES
+from api.utilities.validators.user_validator import check_user
+
+
+@api.route('/registration')
+class Registration(Resource):
+    """
+    Resource class for peforming crud on the asset categories
+    """
+
+    def get(self):
+        return {'status': 'success'}, 200
+
+    def post(self):
+        request_data = request.get_json()
+
+        user_data_schema = UserSchema()
+        user_data = user_data_schema.load_object_into_schema(request_data)
```

Comment:
If you're just loading the data from the request as is, why not use the `load_json_into_schema` method and pass it the `request.data`?